cds_ils/migrator/relations/api.py

```diff
@@ -105,6 +118,53 @@ def migrate_siblings_relation():
                     continue
 
 
+def migrate_series_relations():
+    """Create relations for series."""
+    series_class = current_app_ils.series_record_cls
+    search = search_series_with_relations()
+    results = search.scan()
+
+    for series in results:
+        relations = series["_migration"]["related"]
+
+        for relation in relations:
+            related_series = get_record_by_legacy_recid(
+                series_class, relation["related_recid"]
+            )
+
+            # validate relation type
+            relation_type = Relation.get_relation_by_name(
+                relation["relation_type"]
+            )
+
+            # create relation
+            current_series_record = series_class.get_record_by_pid(series.pid)
+            try:
+                if relation_type in SIBLINGS_RELATION_TYPES:
+                    create_sibling_relation(
+                        current_series_record,
+                        related_series,
+                        relation_type=relation_type,
+                    )
+                elif relation_type in SEQUENCE_RELATION_TYPES:
+                    if relation["sequence_order"] == "previous":
+                        create_sequence_relation(
+                            current_series_record,
+                            related_series,
+                            relation_type=relation_type,
+                        )
+                    else:
+                        create_sequence_relation(
```

Comment:
if the relation is symmetrical (journal A has 780__w pointing to journal B, and journal B has 785__w pointing to journal A) this line (or the 151) will raise an exception because the relation is already created on one of the sides of the relation. Maybe a test would be good for this case?